src/platform/silabs/tracing/SilabsTracing.cpp

```diff
@@ -43,6 +116,28 @@ struct TimeTrackerStorage : public TimeTracker, PersistentData<kPersistentTimeTr
     // TODO implement the Persistent Array class and use it here for logging the watermark array
 };
 
+int TimeTracker::ToCharArray(char * buffer, size_t bufferSize) const
```

Comment:
I think moving this to a switch case would make it easir in the lng temps to add prints.